src/test/cypress/e2e/exam/test-exam/TestExamManagement.cy.ts

```diff
@@ -0,0 +1,166 @@
+            exerciseGroups.visitPageViaUrl(course.id!, exam.id!);
+            exerciseGroups.shouldContainExerciseWithTitle(exerciseGroup.id!, quizExerciseTitle);
+        });
+
+        it('Adds a modeling exercise', () => {
+            cy.visit(`/course-management/${course.id}/exams`);
+            examManagement.openExerciseGroups(exam.id!);
+            exerciseGroups.clickAddModelingExercise(exerciseGroup.id!);
+            const modelingExerciseTitle = 'modeling' + uid;
+            modelingCreation.setTitle(modelingExerciseTitle);
+            modelingCreation.setPoints(10);
+            modelingCreation.save().its('response.statusCode').should('eq', 201);
+            exerciseGroups.visitPageViaUrl(course.id!, exam.id!);
+            exerciseGroups.shouldContainExerciseWithTitle(exerciseGroup.id!, modelingExerciseTitle);
+        });
+
+        it('Adds a programming exercise', () => {
+            cy.visit(`/course-management/${course.id}/exams`);
+            examManagement.openExerciseGroups(exam.id!);
+            exerciseGroups.clickAddProgrammingExercise(exerciseGroup.id!);
+            const programmingExerciseTitle = 'programming' + uid;
+            programmingCreation.setTitle(programmingExerciseTitle);
+            programmingCreation.setShortName(programmingExerciseTitle);
+            programmingCreation.setPackageName('de.test');
+            programmingCreation.setPoints(10);
+            programmingCreation.generate().its('response.statusCode').should('eq', 201);
+            exerciseGroups.visitPageViaUrl(course.id!, exam.id!);
+            exerciseGroups.shouldContainExerciseWithTitle(exerciseGroup.id!, programmingExerciseTitle);
+        });
+
+        it('Edits an exercise group', () => {
+            cy.visit(`/course-management/${course.id}/exams`);
+            examManagement.openExerciseGroups(exam.id!);
+            exerciseGroups.shouldHaveTitle(exerciseGroup.id!, exerciseGroup.title!);
+            exerciseGroups.clickEditGroup(exerciseGroup.id!);
+            const newGroupName = 'Group 3';
+            exerciseGroupCreation.typeTitle(newGroupName);
+            exerciseGroupCreation.update();
+            exerciseGroups.shouldHaveTitle(exerciseGroup.id!, newGroupName);
+            exerciseGroup.title = newGroupName;
+        });
+
+        it('Delete an exercise group', () => {
+            cy.visit('/');
+            navigationBar.openCourseManagement();
+            courseManagement.openExamsOfCourse(course.shortName!);
+            examManagement.openExerciseGroups(exam.id!);
+            // If the group in the "Create group test" was created successfully, we delete it so there is no group with no exercise
+            if (createdGroup) {
+                exerciseGroups.clickDeleteGroup(createdGroup.id!, createdGroup.title!);
+            } else {
+                exerciseGroups.clickDeleteGroup(exerciseGroup.id!, exerciseGroup.title!);
+            }
+        });
```

Comment:
Should there be an assertion which verifies that the group has been deleted?

Reply:
Thanks for the hint, I implemented it in the latest commit 🎉